Dockerfile-oss

```diff
@@ -0,0 +1,115 @@
+# a solid cache layer that only gets reset on pom.xml changes
+RUN mvn -q -ff -T 1C install && rm -rf target
+
+COPY management-api-agent ./management-api-agent
+COPY management-api-common ./management-api-common
+COPY management-api-server ./management-api-server
+COPY management-api-shim-3.x ./management-api-shim-3.x
+COPY management-api-shim-4.x ./management-api-shim-4.x
+RUN mvn -q -ff package -DskipTests
+
+FROM --platform=$BUILDPLATFORM maven:3.6.3-jdk-8-slim as netty4150
+RUN mvn dependency:get -DgroupId=io.netty -DartifactId=netty-all -Dversion=4.1.50.Final -Dtransitive=false
+
+FROM --platform=linux/amd64 cassandra:3.11 as oss311-amd64
+
+FROM --platform=linux/arm64 cassandra:3.11 as oss311-arm64
+# Netty arm64 epoll support was not added until 4.1.50 (https://github.com/netty/netty/pull/9804)
+# Only replace this dependency for arm64 to avoid regressions
+RUN rm /opt/cassandra/lib/netty-all-*.jar
+COPY --from=netty4150 /root/.m2/repository/io/netty/netty-all/4.1.50.Final/netty-all-4.1.50.Final.jar /opt/cassandra/lib/netty-all-4.1.50.Final.jar
+
+FROM oss311-${TARGETARCH} as oss311
+
+ARG TARGETARCH
+
+COPY --from=builder /build/management-api-common/target/datastax-mgmtapi-common-0.1.0-SNAPSHOT.jar /etc/cassandra/
+COPY --from=builder /build/management-api-agent/target/datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar /etc/cassandra/
+COPY --from=builder /build/management-api-server/target/datastax-mgmtapi-server-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
+COPY --from=builder /build/management-api-shim-3.x/target/datastax-mgmtapi-shim-3.x-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
+COPY --from=builder /build/management-api-shim-4.x/target/datastax-mgmtapi-shim-4.x-0.1.0-SNAPSHOT.jar /opt/mgmtapi/
+
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${TARGETARCH} /tini
+RUN chmod +x /tini
+
+RUN set -eux; \
+  rm -fr /etc/apt/sources.list.d/*; \
+  rm -rf /var/lib/apt/lists/*; \
+  apt-get update; \
+  apt-get install -y --no-install-recommends wget iproute2; \
+  rm -rf /var/lib/apt/lists/*
+
+ENV MCAC_VERSION 0.1.7
+ADD https://github.com/datastax/metric-collector-for-apache-cassandra/releases/download/v${MCAC_VERSION}/datastax-mcac-agent-${MCAC_VERSION}.tar.gz /opt/mcac-agent.tar.gz
+RUN mkdir /opt/mcac-agent && tar zxvf /opt/mcac-agent.tar.gz -C /opt/mcac-agent --strip-components 1 && rm /opt/mcac-agent.tar.gz
+
+# backwards compat with upstream ENTRYPOINT
+COPY scripts/docker-entrypoint.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/docker-entrypoint.sh && \
+  ln -sf /usr/local/bin/docker-entrypoint.sh /docker-entrypoint.sh
+
+EXPOSE 9103
+EXPOSE 8080
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
+CMD ["mgmtapi"]
+
+
+# NOTE: Presently, OSS doesn't have official Cassandra 4.0 builds on dockerhub
+# and our build at datastax/cassandra:4.0 is not a multiarch image like the
+# official ones. Once one of those issues is fixed, the following targets can
+# be used to build Cassandra 4.0 with Management API.
```

Comment:
Half way through this I realized we didn't have an appropriate multiarch cassandra 4.0 image. We could just remove all of this for now instead of commenting out. I thought it might be useful for reference, but I have no strong feelings either way.